pxf/Makefile

```diff
@@ -42,6 +42,10 @@ ifneq "$(PXF_VERSION)" ""
     BUILD_PARAMS+= -Pversion="$(PXF_VERSION)"
 endif
 
+ifneq "$(GRADLEW_OPTIONS)" ""
```

Comment:
Another option - add EXCLUDE_PROJECTS, which will take comma-separated list of projects.